common/common.go

```diff
@@ -155,7 +155,7 @@ func YesOrNo(input string) bool {
 func SendHTTPRequest(method, urlPath string, headers map[string]string, body io.Reader) (string, error) {
 	result := strings.ToUpper(method)
 
-	if result != http.MethodPost && result != http.MethodGet && result != http.MethodDelete {
+	if result != http.MethodPost && result != http.MethodGet && result != http.MethodDelete && result != http.MethodPut {
```

Comment:
This check should be done in http.NewRequestWithContext, so can probably rm this entire check for any outbound requests.

Reply:
Bump, this check is actually done via the Go standard library when calling http.NewRequest, so no need for duplicate checks

Reply:
As discussed, can remove this as we're using the Go libraries standard library consts (http.MethodGet etc) and because of the in-built check of methods